File: app/src/main/java/life/plank/juna/zone/data/network/model/SignUpModel.java
package life.plank.juna.zone.data.network.model;

import lombok.Data;

@Data
public class SignUpModel {
    private String objectId;
    private String displayName;
    private String emailAddress;
    private String country;
    private String city;
    private String identityProvider;
    private String givenName;
    private String surname;

    public SignUpModel(String objectId, String displayName, String emailAddress, String country, String city, String identityProvider, String givenName, String surname) {

        this.objectId = objectId;
        this.displayName = displayName;
        this.emailAddress = emailAddress;
        this.country = country;
        this.city = city;
        this.identityProvider = identityProvider;
        this.givenName = givenName;
        this.surname = surname;
    }

}

File: app/src/main/java/life/plank/juna/zone/data/model/user/SignUpModel.kt
package life.plank.juna.zone.data.model.user

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class SignUpModel(
        var objectId: String?,
        var displayName: String?,
        var emailAddress: String?,
        var country: String?,
        var city: String?,
        var identityProvider: String?,
        var givenName: String?,
        var surname: String?
) : Parcelable